integration_tests/src/main/python/ast_test.py

```diff
@@ -258,7 +258,6 @@ def test_lt(data_descr):
             s2 < f.col('b'),
             f.col('a') < f.col('b')))
 
-@datagen_overrides(seed=0, reason='https://github.com/NVIDIA/spark-rapids/issues/9711')
 @pytest.mark.parametrize('data_descr', ast_comparable_descrs, ids=idfn)
 def test_lte(data_descr):
```

Comment:
Like in the other tests. Do we want to explicitly have a NaN Scalar in the tests to verify? Or a dedicated test for it?

Reply:
We're currently generating NaN and null often enough as special cases that this should be covered pretty well across multiple runs.  Want to get these tests fixed for CI, can file followup if desired.